### PR TITLE
Add note that seed should be non-zero

### PR DESCRIPTION
**Pull request by redfast00**
After losing about a half hour debugging why my code wasn't deterministic when it should be, I found out that `randomSeed` doesn't do anything when the `seed` is zero (https://github.com/arduino/ArduinoCore-avr/blob/master/cores/arduino/WMath.cpp#L30)

### CODE REVIEW

**Comment by redfast00** [comment]
Maybe it's useful to have a sort of warning banner/markup instead?

File: Language/Functions/Random Numbers/randomSeed.adoc [hunk 1]
```diff
@@ -21,7 +21,7 @@ subCategories: [ "Random Numbers" ]
 
 If it is important for a sequence of values generated by `random()` to differ, on subsequent executions of a sketch, use randomSeed() to initialize the random number generator with a fairly random input, such as `analogRead()` on an unconnected pin.
 
-Conversely, it can occasionally be useful to use pseudo-random sequences that repeat exactly. This can be accomplished by calling `randomSeed()` with a fixed number, before starting the random sequence.
+Conversely, it can occasionally be useful to use pseudo-random sequences that repeat exactly. This can be accomplished by calling `randomSeed()` with a fixed number, before starting the random sequence. Note that if the `seed` is 0, `randomSeed(seed)` will have no effect.
```
Comment (per1234) [inline Language/Functions/Random Numbers/randomSeed.adoc:24]: >Maybe it's useful to have a sort of warning banner/markup instead?

I agree. Please move this to a "**Notes and Warnings**" section of the page. You can use the source markup of the `random()` reference page as a model:

https://raw.githubusercontent.com/arduino/reference-en/master/Language/Functions/Random%20Numbers/random.adoc

**Review by per1234 — CHANGES_REQUESTED** [review]

**Comment by per1234** [comment]
Related discussion: https://github.com/arduino/Arduino/issues/3887

File: Language/Functions/Random Numbers/randomSeed.adoc (hunk 1 above)
Comment (redfast00) [reply Language/Functions/Random Numbers/randomSeed.adoc]: Good suggestion, thank you!

**Review by per1234 — APPROVED** [review]
Thanks for this valuable improvement @redfast00!

**Pull request merged**